leobot_web_server/scripts/web_server.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nodejs
+
+const static = require('node-static');
+
+const fs = require('fs');
+const path = require('path');
+const rosnodejs = require('rosnodejs');
+
+rosnodejs.initNode('leobot_web_server').then(() => {
+    const nh = rosnodejs.nh;
+
+    console.log('rosnodejs handle: ' + nh);
+
+    nh.getParam('port').then((p) => {
+        const port = p;
+
+        console.log('Port parameter: ' + port);
+    });
+});
+
+const port = 8080; // hard-coded for now
```

Comment:
Please note that all code is located in initNode.then callback https://github.com/RethinkRobotics-opensource/rosnodejs_examples/blob/kinetic-devel/scripts/listener.js#L32-L41